.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: clojure
 lein: 2.7.1
+dist: trusty
+sudo: false
```

Comment:
I didn't see that earlier when I was testing; the `sudo` change was done so that we could use docker containers for faster testing time. Since this key was defined twice I suspect the latter value overwrote the former. I'm going to drop the `sudo: required` bit and see if CI passes.